<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlSetting</class>
 <widget class="QDialog" name="ControlSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>315</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>AR PL UKai CN</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>仪表控制</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../src/image.qrc">
    <normaloff>:/images/leiwo.png</normaloff>:/images/leiwo.png</iconset>
  </property>
  <widget class="QComboBox" name="cbx_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>停止</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>动态展开</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>静态展开</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>收回</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>无动作</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>39</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/setting/UnloaderSetting.png);</string>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_chrs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/parameters/CutHeightResetOn.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>脱开</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>割台提升</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>割台下降</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="m_reset">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/engine/ZeroOn.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;AR PL UKai CN&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>无动作</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>清零</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="auto_load_regu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/con_recong/LoadRegulationAutoOn.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>关闭负荷自动调节</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>开启负荷自动调节</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>无动作</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="auto_height">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/setting/CutHeightAutoOn.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>关闭割台高度自动调节</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>开启割台高度自动调节</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>无动作</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="l_rw_cl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/setting/WeightClearOn.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>无动作</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>清零</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>未定义</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cbx_9">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>无动作</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>静音</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>播放</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="m_reset_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/warning/on.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel_btn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(249, 88, 88);</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OK_btn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>250</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(58, 111, 249);</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="SoftKeyLineEdit" name="reelSpeed">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="SoftKeyLineEdit" name="cutterHz">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>130</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="SoftKeyLineEdit" name="augerSpeed">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>190</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/engine/ReelSpeed.png)</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/images/engine/CutterHz.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/images/engine/FeedAugerSpeed.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>80</y>
     <width>66</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>rpm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>140</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Hz</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>200</y>
     <width>66</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>rpm</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SoftKeyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>softkeylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../src/image.qrc"/>
 </resources>
 <connections/>
</ui>
